client/src/views/Authentication/AuthEmailAction.vue

```diff
@@ -95,10 +95,18 @@ export default {
   },
 
   beforeRouteEnter: (to, from, next) => {
-    if (Authorized.isAuthorized()) {
-      return next('/feed')
-    }
-    next()
+    isAuthorized()
+      .then(res => {
+        if (res === true) {
```

Comment:
return res ? next('/feed') : next() would be shorter and you can remove finally block.